src/components/Proposals/ProposalDetails/ProposalDetails.js

```diff
@@ -105,7 +106,10 @@ function ProposalDetails({ vote }) {
       }
       secondary={
         <>
-          <DisputableActionStatus vote={vote} />
+          <DisputableActionStatus
+            vote={vote}
+            disputableStatus={disputableStatus}
```

Comment:
Why are we passing  this new prop? Are we using it on the component?

Reply:
This is a mistake 🙈